### PR TITLE
[Pavlo Turchyn]final version

### PR DESCRIPTION
**Pull request by FabianMoretti**

### CODE REVIEW

**Comment by FabianMoretti** [comment]
https://fabianmoretti.github.io/ui-internship-lwo2/css-practice/task_2/index.html

File: css-practice/task_2/css/main.css [hunk 1]
```diff
@@ -0,0 +1,1141 @@
...
+header {
+  width: 100%;
+  background: url(../img/header-main.png) no-repeat, url(../img/header-main-bg.png) no-repeat;
+  background-size: cover; }
+
+.header-nav {
+  width: 100%;
+  position: fixed;
+  background: #000000;
+  display: flex;
```
Comment (tarasbobak) [inline css-practice/task_2/css/main.css]: I don't see vendor prefixes in css output

File: css-practice/task_2/css/reset.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
+  margin: 0;
```
Comment (tarasbobak) [inline css-practice/task_2/css/reset.css]: indentation

File: css-practice/task_2/index.html [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+    </div>
+  </header>
+  <section class="bottom-shadow">
+    <div class="brand-bar">
+      <div class="brand-bar-left">
+        <div class="title-main">Social Media</div>
+        <div class="text-main">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet.</div>
+      </div>
+      <div class="brand-bar-right">
+        <div class="facebook icon-size item-hover"></div>
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: aren't those links?

File: css-practice/task_2/index.html [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+          </div>
+          <div class="tab tab-edges-bottom tab-margin tab-hover">
+            <div class="tab-text">TAB 3</div>
+          </div>
+        </div>
+        <div class="features-top-center">
+          <div class="feature-top-title title-main">Tabs with soft transitioning effect.</div>
+          <div class="feature-top-text text-main">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean sodales pulvinar tempor. Cum sociis natoque penatibus et magnis dis parturient montes. <br><br>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet.</div>
+          <div class="features-button">
+            <div class="features-button-name">Download</div>
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: For buttons we have button tag

File: css-practice/task_2/index.html [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+          </div>
+        </div>
+        <div class="features-top-right">
+          <img src="../task_2/img/features-top-right.png" alt="seo" width="100%">
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: why do you set 100% inline for images?

File: css-practice/task_2/index.html [hunk 6]
```diff
@@ -0,0 +1,409 @@
...
+        </div>
+      </div>
+      <div class="features-middle">
+        <div class="middle-main">
+          <div class="middle-left">
+            <img src="./img/features-middle-left.png" alt="seo" width="100%">
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: alt seo? :) you have to write image title there

File: css-practice/task_2/index.html [hunk 7]
```diff
@@ -0,0 +1,409 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="features-bottom">
+        <div class="bottom-main">
+          <div class="bottom-left">
+            <div class="bottom-title title-main">Standard Picture Section</div>
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: you name element title, but don't use heading elements. That's not good

File: css-practice/task_2/index.html [hunk 8]
```diff
@@ -0,0 +1,409 @@
...
+        </div>
+        <div class="footer-wifi item-hover">
+          <img src="img/wifi_small.png" alt="seo" width="100%">
+        </div>
+      </div>
+      <div class="copy-right">© 2015 Axure Themes</div>
+    </div>
+  </footer>
+</body>
+</html>
```
Comment (tarasbobak) [inline css-practice/task_2/index.html]: your markup is a mess of div's. You need to use elements with a meaning.
There are lot's of them, h1-6, p, section, article, main, etc

File: css-practice/task_2/scss/brand-bar.scss [hunk 9]
```diff
@@ -0,0 +1,81 @@
+.brand-bar {
+	width: 78.4%;
+	margin: 0 auto;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+.bottom-shadow {
+	box-shadow: 0 1px 1px $color_12;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/brand-bar.scss]: color_12 is bad name is doesn't tell us anything

File: css-practice/task_2/scss/brand-bar.scss [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+	justify-content: space-between;
+	align-items: center;
+}
+.brand-bar-left {
+	width: 33.4%;
+	margin: 1.8% 0 1.8% 0;
+}
+.title-main {
+	color: $color_2;
+	font-family: $font_family_3;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/brand-bar.scss]: same idea here. That's bad idea of how to name font vars.

File: css-practice/task_2/scss/brand-bar.scss [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+	font-size: 21px;
+	font-weight: bold;
+}
+.text-main {
+	margin-top: 1.5%;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/brand-bar.scss]: Tabs??? For indentation use spaces

File: css-practice/task_2/scss/brand-bar.scss [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+}
+.facebook {
+	background: url(../img/facebook-icon.png) no-repeat;
+	background-size: cover;
+}
+.item-hover {
+	&:hover {
+		opacity: .5;
+		cursor: pointer;
+		opacity: .5;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/brand-bar.scss]: duplication of opacity style

File: css-practice/task_2/scss/brand-bar.scss [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+	background: url(../img/google-icon.png) no-repeat;
+	background-size: cover;
+}
+.pinterest {
+	background: url(../img/pinterest-icon.png) no-repeat;
+	background-size: cover;
+}
+.instagram {
+	margin-right: 1%;
+	background: url(../img/instagram-icon.png) no-repeat;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/brand-bar.scss]: weren't you supposed to use sprites here?
and also there are lot's of duplicate styles. You should have created a common class and reuse it for all of them

File: css-practice/task_2/scss/contact.scss [hunk 14]
```diff
@@ -0,0 +1,128 @@
+.contact {
+	width: 100%;
+	display: flex;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/contact.scss]: i don't see mixins used anywhere

File: css-practice/task_2/scss/veriables.scss [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+$color_5: #86878b;
+$color_6: #212121;
+$color_7: #000000;
+$color_8: #fc791b;
+$color_9: #f5f5f5;
+$color_10: #d6d6d6;
+$color_12: #b8b8b8;
+
+$font_family_1: 'Open Sans - Semibold', sans-serif;
+$font_family_2: 'Raleway', sans-serif;
```
Comment (tarasbobak) [inline css-practice/task_2/scss/veriables.scss]: You mis tabs and spaces, you mix double quotes with single quotes, you don't provide fallback styles for fonts.

**Comment by tarasbobak** [comment]
Mediocre peace of work. Not so bad, not so good.

**Comment by FabianMoretti** [comment]
Дякую

Нд, 18 лист. 2018 20:34 користувач Taras <notifications@github.com> пише:

**Pull request merged**